.github/renovate.json

```diff
@@ -49,6 +62,11 @@
       ],
       "groupName": ["Backstage packages"],
       "dependencyDashboardApproval": true
+    },
+    {
+      "matchDepPatterns": ["^@backstage/"],
```

Comment:
will this affect the plugins we're moving from janus-idp/backstage-plugins to backstage/community-plugins ?

Reply:
yeah it does.  I don't know what the release decision for those upstream plugins will be.  Last I heard, they were going to stay a release behind.

Reply:
@schultzp2020 confirmed this has no impact as the community plugins have a different ns `@backstage-community/*`